server/database/config/db_connection.js

```diff
@@ -0,0 +1,27 @@
+const { Pool } = require('pg');
+require('dotenv').config();
+const url = require('url');
+
+let DB_URL = '';
+if (process.env.NODE_ENV === 'test') {
+  DB_URL = process.env.DATABASE_TESTING;
+} else if (process.env.NODE_ENV === 'dev') {
+  DB_URL = process.env.DATABASE_LOCAL;
+} else if (process.env.NODE_ENV === 'production') {
+  DB_URL = process.env.DATABASE_HEROCO;
+} else {
+  throw new Error('not found DB_URL');
+}
```

Comment:
good job Nariman, but I have a suggestion
try to make switch instead of like :
```
let DB_URL = '';
switch (process.env.NODE_ENV) {
  case 'test': DB_URL = process.env.DATABASE_TESTING; break;
  case 'production': DB_URL = process.env.DATABASE_HEROCO; break;
  case 'development': DB_URL = process.env.DATABASE_LOCAL; break;
  default: throw new Error('not found DB_URL'');
}
```